src/isomorphic/modern/class/__tests__/ReactES6Class-test.js

```diff
@@ -295,23 +279,25 @@ describe('ReactES6Class', () => {
       }
     }
     test(<Foo value="foo" />, 'SPAN', 'foo');
-    expect(lifeCycles).toEqual([
-      'will-mount',
-      'did-mount',
-    ]);
+    expect(lifeCycles).toEqual(['will-mount', 'did-mount']);
     lifeCycles = []; // reset
     test(<Foo value="bar" />, 'SPAN', 'bar');
     expect(lifeCycles).toEqual([
-      'receive-props', freeze({value: 'bar'}),
-      'should-update', freeze({value: 'bar'}), {},
-      'will-update', freeze({value: 'bar'}), {},
-      'did-update', freeze({value: 'foo'}), {},
```

Comment:
fyi, if you have those kind of specially structured data, you can add `// @prettier-ignore` on the line before and prettier will leave it as is.